Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.12.5-slim-bullseye
+
+RUN apt-get update && apt-get install -y git
+
+RUN pip3 install gunicorn
+
+RUN git clone https://github.com/2i2c-org/unnamed-thingity-thing /opt
```

Comment:
I think this should be `COPY . /opt`

Reply:
Hmm, I'm not sure I understand @yuvipanda. Is there anything in the current directory that needs to be copied? I did a `ls` in the container and the repo got cloned in `/opt`. Maybe it should be `git clone https://github.com/2i2c-org/unnamed-thingity-thing /opt/unnamed-thingity-thing` instead?

<img width="745" alt="Screenshot 2024-08-20 at 11 08 04" src="https://github.com/user-attachments/assets/e78ec6da-1b19-43e6-bf8f-1e705b39320f">

Reply:
The answer was here all along :D 


<img width="714" alt="Screenshot 2024-08-20 at 11 14 09" src="https://github.com/user-attachments/assets/35068532-c243-46b6-b5bb-5e5ab2666ebf">

Reply:
The `git clone` will clone the current main, so any builds during PRs won't work. Same for local testing.